src/docs/PML/PMLUserGuide.adoc

```diff
@@ -760,14 +783,9 @@ Set the action `Finish` under Actions and click on `Execute Action`.
 
 image::MAAS_ML_Delete_Service.PNG[align=center]
 
-There are also two other actions that can be executed from Service Automation Portal which are:
-
-- *Update_MaaS_ML*: This action will update the deployed instance according to the updated variables.
-
-- *Pause_MaaS_ML*: This action will pause the service instance.
+There is also two another action that can be executed from Service Automation Portal which is:
```

Comment:
There are also two other actions that can be executed from the Service Automation Portal which are: